tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -172,22 +172,39 @@ test.describe( 'Set up accounts', () => {
 
 			await expect(
 				googleAccountCard.getByRole( 'button', { name: 'Connect' } )
-			).toBeEnabled();
-
-			const mcAccountCard = setUpAccountsPage.getMCAccountCard();
-			await expect(
-				mcAccountCard.getByRole( 'button' )
-			).not.toBeVisible();
-
-			const adsAccountCard = setUpAccountsPage.getGoogleAdsAccountCard();
-			await expect(
-				adsAccountCard.getByRole( 'button' )
-			).not.toBeVisible();
+			).toBeDisabled();
 
 			const continueButton = setUpAccountsPage.getContinueButton();
 			await expect( continueButton ).toBeDisabled();
 		} );
 
+		test( 'should see the terms and conditions checkbox unchecked by default', async () => {
+			const termsCheckbox = setUpAccountsPage.getTermsCheckbox();
+			await expect( termsCheckbox ).not.toBeChecked();
+
+			// Also ensure that connect button is disabled.
+			const connectButton = setUpAccountsPage.getConnectButton();
+			await expect( connectButton ).toBeDisabled();
+		} );
+
+		test( 'should see the connect button and terms and conditions checkbox disabled when jetpack is not connected', async () => {
+			// Mock Jetpack as disconnected
+			await setUpAccountsPage.mockJetpackNotConnected();
+
+			await setUpAccountsPage.goto();
+
+			const connectButton = page
+				.getByRole( 'button', {
+					name: 'Connect',
+					exact: true,
+				} )
+				.nth( 1 );
```

Comment:
Suggest using a more specific way to locate the button.

```suggestion
			const connectButton = setUpAccountsPage
				.getGoogleAccountCard()
				.getByRole( 'button', { name: 'Connect' } );
```

Reply:
Thanks for the suggestion, it seems more appropriate.